features/bootstrap/FeatureContext.php

```diff
@@ -192,9 +192,18 @@ public static function create_cache_dir() {
 	 * @param array $parameters context parameters (set them up through behat.yml)
 	 */
 	public function __construct( array $parameters ) {
+		if ( getenv( 'WP_CLI_TEST_DBUSER' ) ) {
+			self::$db_settings['dbuser'] = getenv( 'WP_CLI_TEST_DBUSER' );
+		}
+
+		if ( getenv( 'WP_CLI_TEST_DBPASS' ) !== false ) {
```

Comment:
Could you use the (unnatural, hard to read & pointless) "yoda" style here to conform to WPCS?